ui/src/components/header/UserProfile.tsx

```diff
@@ -45,24 +43,13 @@ export default function UserProfile(props: UserProfileProps) {
         login =
           metadata[authMethodPreferredUsernameKey as keyof typeof metadata];
       }
-
-      const authMethodIssuerKey = `io.flipt.auth.${authMethod}.issuer`;
-      if (metadata[authMethodIssuerKey as keyof typeof metadata]) {
-        const logoutURI =
-          metadata[authMethodIssuerKey as keyof typeof metadata];
-        logoutURL = `//${logoutURI}`;
-      }
```

Comment:
@markphelps are there any potential consequences for Flipt Cloud taking into account that user will be redirected to the login page?

Reply:
yeah, ideally we should detect if the issuer is flipt cloud, and if so redirect there. if not then we should redirect to `/login`